ui/core/components/inputs/consumables.ts

```diff
@@ -519,81 +502,69 @@ export const PetScrollOfStrengthV = makeBooleanConsumeInput({
-export const PotionOfWildMagic = {
-	actionId: ActionId.fromItemId(40212),
-	value: Potions.PotionOfWildMagic,
+
+export const MythicalHealingPotion = {
+	actionId: ActionId.fromItemId(57191),
+	value: Potions.MythicalHealingPotion,
 };
 
-export const DestructionPotion = {
-	actionId: ActionId.fromItemId(22839),
-	value: Potions.DestructionPotion,
+export const MythicalManaPotion = {
+	actionId: ActionId.fromItemId(57192),
+	value: Potions.MythicalManaPotion,
 };
+export const PotionOfSpeed = { actionId: ActionId.fromItemId(40211), value: Potions.PotionOfSpeed };
 export const HastePotion = { actionId: ActionId.fromItemId(22838), value: Potions.HastePotion };
 export const MightyRagePotion = {
 	actionId: ActionId.fromItemId(13442),
 	value: Potions.MightyRagePotion,
 };
-export const SuperManaPotion = {
-	actionId: ActionId.fromItemId(22832),
-	value: Potions.SuperManaPotion,
-};
-export const FelManaPotion = { actionId: ActionId.fromItemId(31677), value: Potions.FelManaPotion };
-export const InsaneStrengthPotion = {
-	actionId: ActionId.fromItemId(22828),
-	value: Potions.InsaneStrengthPotion,
-};
-export const IronshieldPotion = {
-	actionId: ActionId.fromItemId(22849),
-	value: Potions.IronshieldPotion,
-};
-export const HeroicPotion = { actionId: ActionId.fromItemId(22837), value: Potions.HeroicPotion };
 
 export const POTIONS_CONFIG = [
-	{ config: RunicHealingPotion, stats: [Stat.StatStamina] },
-	{ config: RunicHealingInjector, stats: [Stat.StatStamina] },
-	{ config: RunicManaPotion, stats: [Stat.StatIntellect] },
-	{ config: RunicManaInjector, stats: [Stat.StatIntellect] },
-	{ config: IndestructiblePotion, stats: [Stat.StatArmor] },
-	{ config: InsaneStrengthPotion, stats: [Stat.StatStrength] },
-	{ config: HeroicPotion, stats: [Stat.StatStamina] },
+	{ config: GolembloodPotion, stats: [Stat.StatStrength] },
+	{ config: PotionOfTheTolvir, stats: [Stat.StatAgility] },
+	{ config: PotionOfConcentration, stats: [Stat.StatMana] },
+	{ config: VolcanicPotion, stats: [Stat.StatIntellect] },
+	{ config: EarthenPotion, stats: [Stat.StatArmor] },
+	{ config: MightyRejuvenationPotion, stats: [Stat.StatMana, Stat.StatHealth] },
+	{ config: MythicalHealingPotion, stats: [Stat.StatHealth] },
+	{ config: MythicalManaPotion, stats: [Stat.StatMana] },
 	{ config: PotionOfSpeed, stats: [Stat.StatMeleeHaste, Stat.StatSpellHaste] },
-	{
-		config: PotionOfWildMagic,
-		stats: [Stat.StatMeleeCrit, Stat.StatSpellCrit, Stat.StatSpellPower],
-	},
 ] as ConsumableStatOption<Potions>[];
 
 export const PRE_POTIONS_CONFIG = [
-	{ config: IndestructiblePotion, stats: [Stat.StatArmor] },
-	{ config: InsaneStrengthPotion, stats: [Stat.StatStrength] },
-	{ config: HeroicPotion, stats: [Stat.StatStamina] },
+	{ config: GolembloodPotion, stats: [Stat.StatStrength] },
+	{ config: PotionOfTheTolvir, stats: [Stat.StatAgility] },
+	{ config: VolcanicPotion, stats: [Stat.StatIntellect] },
```

Comment:
Why are not all potions added as an option in the pre-pot? I dont think we want to assume what the user wants to do. If they want to prepot a potion that doesnt make sense for us they should still be able to do so

Reply:
The only potion not in that list that makes sense to add is Earthen Potion, everything else is mana or health gain. But in that case, why even have separate configs?
I can just do
`export const PRE_POTIONS_CONFIG = [
	...POTIONS_CONFIG
] as ConsumableStatOption<Potions>[]; or refactor both to use the same config.
`

Reply:
I just added the earthen pot for now, I dont think it makes sense at all to have mana and health pots as prepots

Reply:
Yeah. Mana/Health is not needed indeed and its better if they dont clutter the ui for the prepot then